koku/reporting/provider/gcp/sql/views/reporting_gcp_database_summary.sql

```diff
@@ -6,12 +6,14 @@ CREATE MATERIALIZED VIEW reporting_gcp_database_summary AS(
         usage_start,
         usage_start as usage_end,
         account_id,
-        sum(usage_amount) as usage_amount,
+        SUM(case when usage_amount = 'NaN' then 0.0::numeric(24,9) else usage_amount end::numeric(24,9)) AS usage_amount,
```

Comment:
I think we should move this case to the daily summary table. We'll fall back to that table when the query can't be satisfied by the view. So that way in daily summary usage amount will always be a numeric, and you can just leave the views as-is.